### PR TITLE
Add Hugo theme

### PR DESCRIPTION
**Pull request by cmaureir**

### CODE REVIEW

**Comment by Mariatta** [comment]
Please review @DennyPerez18 @mjmolina

**Comment by Mariatta** [comment]
@cmaureir Can we set up netlify preview?

File: content/_index.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+![About Icon class=icon-section](/img/about-icon.png)
+
+{{< pinktitle "About the conference" >}}
+
+Attention all PyLadies community members! We’re excited to share that we are in
+the early stages of planning a PyLadies Conference (PyLadiesCon),
+a transformative event designed to promote diversity, learning, and empowerment
+within the Python community.  🎉
+
+Save the date! The conference will take place on the first weekend of December,
```
Comment (cmaureir) [inline content/_index.md:49]:
```suggestion
Save the date! The conference will take place on December 6th-8th,
```

File: content/_index.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+            <i class="fas fa-quote-left fa-xs pink" aria-hidden="true"></i>
+            <span>The more I study, the more insatiable do I feel my genius for it to be.</span>
+            <i class="fas fa-quote-right fa-xs pink" aria-hidden="true"></i>
+          </p>
+          <footer class="blockquote-footer">Ada Lovelace</footer>
+        </blockquote>
+        <h1 class="pink display-4 fw-normal">PyLadiesCon</h1>
+        <h4 class="gradient special-font">2024 Online</h4>
+        <p class="lead fw-normal">
+          TBD December | Multi-language | Multi-timezone
```
Comment (cmaureir) [inline content/_index.md:23]:
```suggestion
          6th-8th December | Multi-language | Multi-timezone
```

File: content/about/_index.md [hunk 3]
```diff
@@ -0,0 +1,369 @@
...
+
+{{% columns %}}
+
+## The Event
+
+PyLadies Conference (PyLadiesCon) is the first Pyladies conference
+held– an exciting online and FREE event dedicated to empowerment,
+learning, and diversity within the Python community! 🎉
+
+🗓️ Date TBD
```
Comment (cmaureir) [inline content/about/_index.md:34]:
```suggestion
🗓️ Date: December 6th-8th
```

File: content/sponsors/_index.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
++++
+title = 'Sponsors'
+draft = false
+layout = 3
++++
+
+{{< pageheadersmall
+  "Sponsor Prospectus for the global PyLadies Conference"
+  "Join us online across the globe, Dec TBD, 2024!"
```
Comment (cmaureir) [inline content/sponsors/_index.md:9]:
```suggestion
  "Join us online across the globe, Dec 6th-8th, 2024!"
```

File: content/_index.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+    <div class="d-md-flex flex-md-equal w-75 my-md-3 mx-auto align-items-center">
+      <div class="col-md-5 p-lg-3 mx-auto my-5">
+        <blockquote class="blockquote text-center">
+          <p>
+            <i class="fas fa-quote-left fa-xs pink" aria-hidden="true"></i>
+            <span>The more I study, the more insatiable do I feel my genius for it to be.</span>
+            <i class="fas fa-quote-right fa-xs pink" aria-hidden="true"></i>
+          </p>
+          <footer class="blockquote-footer">Ada Lovelace</footer>
+        </blockquote>
```
Comment (cmaureir) [inline content/_index.md:19]:
```suggestion
```

File: content/about/_index.md [hunk 6]
```diff
@@ -0,0 +1,369 @@
...
+gatherings.
+
+To learn more,
+{{< rarrowlinkwhite "Visit The PyLadies Website" "https://pyladies.com" >}}
+
+## The Volunteers
+
+Thanks to all the people that submitted their interest into helping
+PyLadiesCon. Currently, we have enough helpers to make this an amazing
+conference. If a new Volunteer Call happens, we will everyone.
```
Comment (Mariatta) [inline content/about/_index.md:78]:
```suggestion
conference. If a new Volunteer Call happens, we will contact everyone.
```

File: content/sponsors/_index.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+To learn more and to join us as a sponsor, please contact
+[sponsors@pyladies.com](mailto:sponsors@pyladies.com), and a member of our
+sponsorship team will get back to you soon.
+{{< rarrowlinkwhite "Get in touch" "mailto:sponsors@pyladies.com" >}}
+
+## The Volunteers
+
+Thanks to all the people that submitted their interest into helping
+PyLadiesCon. Currently, we have enough helpers to make this an amazing
+conference. If a new Volunteer Call happens, we will everyone.
```
Comment (Mariatta) [inline content/sponsors/_index.md:51]:
```suggestion
conference. If a new Volunteer Call happens, we will contact everyone.
```

File: themes/pyladies/LICENSE [hunk 8]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) [year] [fullname]
```
Comment (Mariatta) [inline themes/pyladies/LICENSE:3]:
```suggestion
Copyright (c) 2024 PyLadies Global Conference
```

**Pull request merged**